From c2b5492ae92048636f724838e3c079c3b7584fc2 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Thu, 11 Oct 2018 11:34:31 +0530
Subject: [PATCH 0464/1851] drm: xlnx: Select VIDEOMODE_HELPERS when
 DRM_XLNX_PL_DISP is selected

commit f49ac0aa7c75b2c14621b36814c4b1197f5c0117 from
https://github.com/Xilinx/linux-xlnx.git

Select the VIDEOMODE_HELPERS when DRM_XLNX_PL_DISP is selected in
menuconfig. This fixes the compilation error of undefined reference to
drm_display_mode_to_videomode in the Xilinx PL Display driver.

This is also added to DRM_XLNX_MIXER as the same API is being used
there.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/xlnx/Kconfig b/drivers/gpu/drm/xlnx/Kconfig
index 033f18402492..f220ccad46f0 100644
--- a/drivers/gpu/drm/xlnx/Kconfig
+++ b/drivers/gpu/drm/xlnx/Kconfig
@@ -57,12 +57,14 @@ config DRM_XLNX_DSI
 config DRM_XLNX_MIXER
 	tristate "Xilinx DRM Mixer Driver"
 	depends on DRM_XLNX
+	select VIDEOMODE_HELPERS
 	help
 	  DRM driver for Xilinx Mixer driver
 
 config DRM_XLNX_PL_DISP
 	tristate "Xilinx DRM PL display driver"
 	depends on DRM_XLNX
+	select VIDEOMODE_HELPERS
 	help
 	  DRM driver for Xilinx PL display driver, provides drm
 	  crtc and plane object to display pipeline. You need to
-- 
2.31.1

